src/constants/layout.js

```diff
@@ -0,0 +1 @@
+export const heroVideoRatio = 2.134
```

Comment:
please remove this file and move this value to the VideoPlayer component because it's a local variable. I should do this on shinta project too.